helix-core/src/main/java/org/apache/helix/messaging/handling/HelixTaskExecutor.java

```diff
@@ -488,11 +490,11 @@ public boolean scheduleTask(MessageTask task) {
               "Message handling task already sheduled for " + taskId, manager);
         }
       }
-    } catch (Exception e) {
+    } catch (Throwable e) {
```

Comment:
In Helix, we use general Exception instead of Throwable. Even Throwable, you need to replace variable name to t.